<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="configPathLabel">
        <property name="text">
         <string>Config-Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="configPathLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="configPathToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="activeBox">
          <property name="text">
           <string>Active</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="hasParentWindowLabel">
        <property name="text">
         <string>Has Parent Window:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="hasParentWindowCheckBox">
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="displayLevelLabel">
        <property name="text">
         <string>DisplayLevel:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="displayLevelComboBox">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>AutomaticLevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ExitLevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InfoLevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ExtendedInfoLevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ProgressLevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AskLevel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="scheduleUpdateDateCheckBox">
        <property name="text">
         <string>Schedule Update:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateTimeEdit" name="scheduleUpdateDateTimeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <layout class="QVBoxLayout" name="buttonLayout">
        <item>
         <widget class="QCommandLinkButton" name="checkUpdatesButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Check for Updates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Send cancel request</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="detailedInfosLabel">
        <property name="text">
         <string>Detailed Infos</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="detailedInfosCheckBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>configPathToolButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkUpdatesButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cancelButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>configPathLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scheduleUpdateDateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>scheduleUpdateDateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hasParentWindowCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>detailedInfosCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
